backend/routes/news.js

```diff
@@ -17,29 +17,43 @@ router.get("/", async (req, res) => {
   }
 });
 
+// user does not access this? For now, will access when user input is considered in seeded data (maybe?)
```

Comment:
yea for now seed news seems like something that an async / cron job would occasionally run. We could have a seperate endpoint for a user-specific stuff which reads from our database